reference/config_files/global_conf.rst

```diff
@@ -183,7 +211,7 @@ Networking confs
 .. _reference_config_files_global_conf_ssl_certificates:
 
 Configuration of client certificates
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+++++++++++++++++++++++++++++++++++++
```

Comment:
Had to change this to be consistent with header fromatting (it was an error otherwise)